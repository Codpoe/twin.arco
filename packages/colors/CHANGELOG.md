# @twin.arco/colors

## 0.1.0
### Minor Changes

- Feature: first release
