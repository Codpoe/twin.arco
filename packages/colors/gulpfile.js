require('esbuild-register');

const { arco } = require('./src/utils/gulp.ts');

exports.arco = arco;
