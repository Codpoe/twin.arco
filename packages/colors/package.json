{
  "name": "@twin.arco/colors",
  "version": "0.1.1",
  "jsnext:source": "./src/index.ts",
  "types": "./dist/types/index.d.ts",
  "main": "./dist/js/node/index.js",
  "module": "./dist/js/treeshaking/index.js",
  "jsnext:modern": "./dist/js/modern/index.js",
  "scripts": {
    "prepare": "modern build",
    "prepublishOnly": "modern build --platform",
    "dev": "modern dev",
    "build": "modern build",
    "build:arco": "gulp arco",
    "prebuild": "npm run build:arco",
    "new": "modern new",
    "test": "modern test --passWithNoTests"
  },
  "dependencies": {
    "@babel/runtime": "^7.16.3"
  },
  "devDependencies": {
    "@arco-design/web-react": "^2.25.1",
    "@modern-js/module-tools": "^1.1.1",
    "@modern-js/plugin-jarvis": "^1.1.1",
    "@modern-js/plugin-testing": "^1.1.1",
    "@types/gulp": "^4.0.9",
    "@types/gulp-clean-css": "^4.3.0",
    "@types/gulp-less": "^0.0.32",
    "@types/gulp-postcss": "^8.0.1",
    "@types/jest": "^26.0.24",
    "@types/node": "^14.17.34",
    "@types/through2": "^2.0.36",
    "esbuild": "^0.13.15",
    "esbuild-register": "^3.2.0",
    "gulp": "^4.0.2",
    "gulp-clean-css": "^4.3.0",
    "gulp-less": "^5.0.0",
    "gulp-postcss": "^9.0.1",
    "less": "^4.1.2",
    "plugin-error": "^1.0.1",
    "postcss": "^8.4.4",
    "postcss-js": "^3.0.3",
    "postcss-values-parser": "^6.0.1",
    "react": "^17.0.2",
    "through2": "^4.0.2",
    "ts-morph": "^12.2.0",
    "typescript": "^4.5.2"
  },
  "sideEffects": [
    "*.css",
    "*.less",
    "*.sass",
    "*.scss"
  ],
  "publishConfig": {
    "access": "public",
    "registry": "https://registry.npmjs.org/"
  }
}
