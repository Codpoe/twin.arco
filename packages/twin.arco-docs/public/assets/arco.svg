<svg width="64" height="64" viewBox="0 0 64 64" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0_8748:93584)">
<path fill-rule="evenodd" clip-rule="evenodd" d="M9.75508 32.9591L24.4996 17.8607C28.4325 13.8334 34.8855 13.7569 38.9128 17.6898C38.9704 17.7461 39.0274 17.803 39.0837 17.8607L39.2442 18.025C43.2099 22.0859 43.2099 28.5696 39.2442 32.6304L24.4996 47.7289C20.5667 51.7561 14.1137 51.8327 10.0864 47.8998C10.0288 47.8435 9.97183 47.7865 9.91554 47.7289L9.75508 47.5645C5.78935 43.5036 5.78935 37.02 9.75508 32.9591Z" fill="#12D2AC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M39.0958 17.8608L53.6798 32.7948C57.7348 36.9471 57.7348 43.5767 53.6798 47.7289C49.7469 51.7562 43.2939 51.8327 39.2667 47.8998C39.209 47.8436 39.1521 47.7866 39.0958 47.7289L24.5117 32.7948C20.4567 28.6426 20.4567 22.013 24.5117 17.8608C28.4446 13.8335 34.8976 13.757 38.9249 17.6898C38.9825 17.7461 39.0395 17.8031 39.0958 17.8608Z" fill="#307AF2"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M39.2641 18.0433L46.3702 25.3199L31.7861 40.254L17.2021 25.3199L24.3081 18.0433C28.3413 13.9133 34.9589 13.8349 39.0889 17.8681C39.148 17.9258 39.2064 17.9842 39.2641 18.0433Z" fill="#0057FE"/>
</g>
<defs>
<clipPath id="clip0_8748:93584">
<rect width="52" height="38" fill="white" transform="translate(6 13)"/>
</clipPath>
</defs>
</svg>
