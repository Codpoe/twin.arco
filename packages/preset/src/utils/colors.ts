import { palette, PaletteName, theme, ThemeName } from '@twin.arco/colors';
import {
  cssRgb,
  cssRgba,
  cssVar,
  defaultKeyFormatter,
  normalizeCssVariable,
} from './formatter';

export const themeKeys: ThemeName[] = theme.map(normalizeCssVariable);

export const paletteKeys: PaletteName[] = palette.map(normalizeCssVariable);

export function createTheme(
  formatter: (value: `--${string}`) => string = defaultKeyFormatter,
) {
  return theme.reduce((colors, key) => {
    const _key = formatter(key);
    colors[_key] = cssRgb(cssVar(key));
    return colors;
  }, {} as Record<string, any>);
}

export function createPalette(
  opacity = true,
  formatter: (value: `--${string}`) => string = defaultKeyFormatter,
) {
  return palette.reduce((colors, key) => {
    const _key = formatter(key);
    if (!opacity) {
      colors[_key] = cssRgb(cssVar(key));
    } else {
      colors[_key] = colorWithOpacity(cssVar(key));
    }
    return colors;
  }, {} as Record<string, any>);
}

export function colorWithOpacity(key: string) {
  return ({ opacityVariable }: { [key: string]: string }) =>
    cssRgba(key, cssVar(opacityVariable as `--${string}`, '1'));
}
