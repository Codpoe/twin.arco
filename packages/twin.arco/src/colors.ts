import { theme, palette, ThemeName, PaletteName } from '@twin.arco/colors';
import { normalizeCSSVariable } from './utils';

export const themeKeys: ThemeName[] = theme.map(normalizeCSSVariable);

export const paletteKeys: PaletteName[] = palette.map(normalizeCSSVariable);
