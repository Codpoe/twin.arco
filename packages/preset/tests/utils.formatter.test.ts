import { theme, palette } from '@twin.arco/colors';
import {
  cssRgb,
  normalizeCssVariable,
  cssRgba,
  cssVar,
  defaultKeyFormatter,
} from '@/utils';

describe('Utils formatter', () => {
  test('normalCssVariable', () => {
    expect(normalizeCssVariable('--red-1')).toBe('red-1');
  });

  test('cssVar', () => {
    expect(cssVar('--red-1')).toBe('var(--red-1)');
    expect(cssVar('--red-1', 'red')).toBe('var(--red-1, red)');
  });

  test('cssRgb', () => {
    expect(cssRgb('test')).toBe('rgb(test)');
  });

  test('cssRgba', () => {
    expect(cssRgba('test')).toBe('rgba(test, 1)');
    expect(cssRgba('test', 0.5)).toBe('rgba(test, 0.5)');
  });

  test('defaultKeyFormatter', () => {
    theme.forEach(key => {
      expect(defaultKeyFormatter(key)).toMatchSnapshot();
    });

    palette.forEach(key => {
      expect(defaultKeyFormatter(key)).toMatchSnapshot();
    });
  });
});
