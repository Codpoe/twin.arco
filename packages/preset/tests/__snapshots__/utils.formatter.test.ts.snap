// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils formatter defaultKeyFormatter 1`] = `"arco-white"`;

exports[`Utils formatter defaultKeyFormatter 2`] = `"arco-black"`;

exports[`Utils formatter defaultKeyFormatter 3`] = `"arco-border"`;

exports[`Utils formatter defaultKeyFormatter 4`] = `"arco-bg-popup"`;

exports[`Utils formatter defaultKeyFormatter 5`] = `"arco-bg-1"`;

exports[`Utils formatter defaultKeyFormatter 6`] = `"arco-bg-2"`;

exports[`Utils formatter defaultKeyFormatter 7`] = `"arco-bg-3"`;

exports[`Utils formatter defaultKeyFormatter 8`] = `"arco-bg-4"`;

exports[`Utils formatter defaultKeyFormatter 9`] = `"arco-bg-5"`;

exports[`Utils formatter defaultKeyFormatter 10`] = `"arco-bg-white"`;

exports[`Utils formatter defaultKeyFormatter 11`] = `"arco-neutral-1"`;

exports[`Utils formatter defaultKeyFormatter 12`] = `"arco-neutral-2"`;

exports[`Utils formatter defaultKeyFormatter 13`] = `"arco-neutral-3"`;

exports[`Utils formatter defaultKeyFormatter 14`] = `"arco-neutral-4"`;

exports[`Utils formatter defaultKeyFormatter 15`] = `"arco-neutral-5"`;

exports[`Utils formatter defaultKeyFormatter 16`] = `"arco-neutral-6"`;

exports[`Utils formatter defaultKeyFormatter 17`] = `"arco-neutral-7"`;

exports[`Utils formatter defaultKeyFormatter 18`] = `"arco-neutral-8"`;

exports[`Utils formatter defaultKeyFormatter 19`] = `"arco-neutral-9"`;

exports[`Utils formatter defaultKeyFormatter 20`] = `"arco-neutral-10"`;

exports[`Utils formatter defaultKeyFormatter 21`] = `"arco-text-1"`;

exports[`Utils formatter defaultKeyFormatter 22`] = `"arco-text-2"`;

exports[`Utils formatter defaultKeyFormatter 23`] = `"arco-text-3"`;

exports[`Utils formatter defaultKeyFormatter 24`] = `"arco-text-4"`;

exports[`Utils formatter defaultKeyFormatter 25`] = `"arco-border-1"`;

exports[`Utils formatter defaultKeyFormatter 26`] = `"arco-border-2"`;

exports[`Utils formatter defaultKeyFormatter 27`] = `"arco-border-3"`;

exports[`Utils formatter defaultKeyFormatter 28`] = `"arco-border-4"`;

exports[`Utils formatter defaultKeyFormatter 29`] = `"arco-fill-1"`;

exports[`Utils formatter defaultKeyFormatter 30`] = `"arco-fill-2"`;

exports[`Utils formatter defaultKeyFormatter 31`] = `"arco-fill-3"`;

exports[`Utils formatter defaultKeyFormatter 32`] = `"arco-fill-4"`;

exports[`Utils formatter defaultKeyFormatter 33`] = `"arco-primary-light-1"`;

exports[`Utils formatter defaultKeyFormatter 34`] = `"arco-primary-light-2"`;

exports[`Utils formatter defaultKeyFormatter 35`] = `"arco-primary-light-3"`;

exports[`Utils formatter defaultKeyFormatter 36`] = `"arco-primary-light-4"`;

exports[`Utils formatter defaultKeyFormatter 37`] = `"arco-secondary"`;

exports[`Utils formatter defaultKeyFormatter 38`] = `"arco-secondary-hover"`;

exports[`Utils formatter defaultKeyFormatter 39`] = `"arco-secondary-active"`;

exports[`Utils formatter defaultKeyFormatter 40`] = `"arco-secondary-disabled"`;

exports[`Utils formatter defaultKeyFormatter 41`] = `"arco-danger-light-1"`;

exports[`Utils formatter defaultKeyFormatter 42`] = `"arco-danger-light-2"`;

exports[`Utils formatter defaultKeyFormatter 43`] = `"arco-danger-light-3"`;

exports[`Utils formatter defaultKeyFormatter 44`] = `"arco-danger-light-4"`;

exports[`Utils formatter defaultKeyFormatter 45`] = `"arco-success-light-1"`;

exports[`Utils formatter defaultKeyFormatter 46`] = `"arco-success-light-2"`;

exports[`Utils formatter defaultKeyFormatter 47`] = `"arco-success-light-3"`;

exports[`Utils formatter defaultKeyFormatter 48`] = `"arco-success-light-4"`;

exports[`Utils formatter defaultKeyFormatter 49`] = `"arco-warning-light-1"`;

exports[`Utils formatter defaultKeyFormatter 50`] = `"arco-warning-light-2"`;

exports[`Utils formatter defaultKeyFormatter 51`] = `"arco-warning-light-3"`;

exports[`Utils formatter defaultKeyFormatter 52`] = `"arco-warning-light-4"`;

exports[`Utils formatter defaultKeyFormatter 53`] = `"arco-link-light-1"`;

exports[`Utils formatter defaultKeyFormatter 54`] = `"arco-link-light-2"`;

exports[`Utils formatter defaultKeyFormatter 55`] = `"arco-link-light-3"`;

exports[`Utils formatter defaultKeyFormatter 56`] = `"arco-link-light-4"`;

exports[`Utils formatter defaultKeyFormatter 57`] = `"arco-tooltip-bg"`;

exports[`Utils formatter defaultKeyFormatter 58`] = `"arco-spin-layer-bg"`;

exports[`Utils formatter defaultKeyFormatter 59`] = `"arco-menu-dark-bg"`;

exports[`Utils formatter defaultKeyFormatter 60`] = `"arco-menu-light-bg"`;

exports[`Utils formatter defaultKeyFormatter 61`] = `"arco-menu-dark-hover"`;

exports[`Utils formatter defaultKeyFormatter 62`] = `"arco-mask-bg"`;

exports[`Utils formatter defaultKeyFormatter 63`] = `"red-1"`;

exports[`Utils formatter defaultKeyFormatter 64`] = `"red-2"`;

exports[`Utils formatter defaultKeyFormatter 65`] = `"red-3"`;

exports[`Utils formatter defaultKeyFormatter 66`] = `"red-4"`;

exports[`Utils formatter defaultKeyFormatter 67`] = `"red-5"`;

exports[`Utils formatter defaultKeyFormatter 68`] = `"red-6"`;

exports[`Utils formatter defaultKeyFormatter 69`] = `"red-7"`;

exports[`Utils formatter defaultKeyFormatter 70`] = `"red-8"`;

exports[`Utils formatter defaultKeyFormatter 71`] = `"red-9"`;

exports[`Utils formatter defaultKeyFormatter 72`] = `"red-10"`;

exports[`Utils formatter defaultKeyFormatter 73`] = `"orangered-1"`;

exports[`Utils formatter defaultKeyFormatter 74`] = `"orangered-2"`;

exports[`Utils formatter defaultKeyFormatter 75`] = `"orangered-3"`;

exports[`Utils formatter defaultKeyFormatter 76`] = `"orangered-4"`;

exports[`Utils formatter defaultKeyFormatter 77`] = `"orangered-5"`;

exports[`Utils formatter defaultKeyFormatter 78`] = `"orangered-6"`;

exports[`Utils formatter defaultKeyFormatter 79`] = `"orangered-7"`;

exports[`Utils formatter defaultKeyFormatter 80`] = `"orangered-8"`;

exports[`Utils formatter defaultKeyFormatter 81`] = `"orangered-9"`;

exports[`Utils formatter defaultKeyFormatter 82`] = `"orangered-10"`;

exports[`Utils formatter defaultKeyFormatter 83`] = `"orange-1"`;

exports[`Utils formatter defaultKeyFormatter 84`] = `"orange-2"`;

exports[`Utils formatter defaultKeyFormatter 85`] = `"orange-3"`;

exports[`Utils formatter defaultKeyFormatter 86`] = `"orange-4"`;

exports[`Utils formatter defaultKeyFormatter 87`] = `"orange-5"`;

exports[`Utils formatter defaultKeyFormatter 88`] = `"orange-6"`;

exports[`Utils formatter defaultKeyFormatter 89`] = `"orange-7"`;

exports[`Utils formatter defaultKeyFormatter 90`] = `"orange-8"`;

exports[`Utils formatter defaultKeyFormatter 91`] = `"orange-9"`;

exports[`Utils formatter defaultKeyFormatter 92`] = `"orange-10"`;

exports[`Utils formatter defaultKeyFormatter 93`] = `"gold-1"`;

exports[`Utils formatter defaultKeyFormatter 94`] = `"gold-2"`;

exports[`Utils formatter defaultKeyFormatter 95`] = `"gold-3"`;

exports[`Utils formatter defaultKeyFormatter 96`] = `"gold-4"`;

exports[`Utils formatter defaultKeyFormatter 97`] = `"gold-5"`;

exports[`Utils formatter defaultKeyFormatter 98`] = `"gold-6"`;

exports[`Utils formatter defaultKeyFormatter 99`] = `"gold-7"`;

exports[`Utils formatter defaultKeyFormatter 100`] = `"gold-8"`;

exports[`Utils formatter defaultKeyFormatter 101`] = `"gold-9"`;

exports[`Utils formatter defaultKeyFormatter 102`] = `"gold-10"`;

exports[`Utils formatter defaultKeyFormatter 103`] = `"yellow-1"`;

exports[`Utils formatter defaultKeyFormatter 104`] = `"yellow-2"`;

exports[`Utils formatter defaultKeyFormatter 105`] = `"yellow-3"`;

exports[`Utils formatter defaultKeyFormatter 106`] = `"yellow-4"`;

exports[`Utils formatter defaultKeyFormatter 107`] = `"yellow-5"`;

exports[`Utils formatter defaultKeyFormatter 108`] = `"yellow-6"`;

exports[`Utils formatter defaultKeyFormatter 109`] = `"yellow-7"`;

exports[`Utils formatter defaultKeyFormatter 110`] = `"yellow-8"`;

exports[`Utils formatter defaultKeyFormatter 111`] = `"yellow-9"`;

exports[`Utils formatter defaultKeyFormatter 112`] = `"yellow-10"`;

exports[`Utils formatter defaultKeyFormatter 113`] = `"lime-1"`;

exports[`Utils formatter defaultKeyFormatter 114`] = `"lime-2"`;

exports[`Utils formatter defaultKeyFormatter 115`] = `"lime-3"`;

exports[`Utils formatter defaultKeyFormatter 116`] = `"lime-4"`;

exports[`Utils formatter defaultKeyFormatter 117`] = `"lime-5"`;

exports[`Utils formatter defaultKeyFormatter 118`] = `"lime-6"`;

exports[`Utils formatter defaultKeyFormatter 119`] = `"lime-7"`;

exports[`Utils formatter defaultKeyFormatter 120`] = `"lime-8"`;

exports[`Utils formatter defaultKeyFormatter 121`] = `"lime-9"`;

exports[`Utils formatter defaultKeyFormatter 122`] = `"lime-10"`;

exports[`Utils formatter defaultKeyFormatter 123`] = `"green-1"`;

exports[`Utils formatter defaultKeyFormatter 124`] = `"green-2"`;

exports[`Utils formatter defaultKeyFormatter 125`] = `"green-3"`;

exports[`Utils formatter defaultKeyFormatter 126`] = `"green-4"`;

exports[`Utils formatter defaultKeyFormatter 127`] = `"green-5"`;

exports[`Utils formatter defaultKeyFormatter 128`] = `"green-6"`;

exports[`Utils formatter defaultKeyFormatter 129`] = `"green-7"`;

exports[`Utils formatter defaultKeyFormatter 130`] = `"green-8"`;

exports[`Utils formatter defaultKeyFormatter 131`] = `"green-9"`;

exports[`Utils formatter defaultKeyFormatter 132`] = `"green-10"`;

exports[`Utils formatter defaultKeyFormatter 133`] = `"cyan-1"`;

exports[`Utils formatter defaultKeyFormatter 134`] = `"cyan-2"`;

exports[`Utils formatter defaultKeyFormatter 135`] = `"cyan-3"`;

exports[`Utils formatter defaultKeyFormatter 136`] = `"cyan-4"`;

exports[`Utils formatter defaultKeyFormatter 137`] = `"cyan-5"`;

exports[`Utils formatter defaultKeyFormatter 138`] = `"cyan-6"`;

exports[`Utils formatter defaultKeyFormatter 139`] = `"cyan-7"`;

exports[`Utils formatter defaultKeyFormatter 140`] = `"cyan-8"`;

exports[`Utils formatter defaultKeyFormatter 141`] = `"cyan-9"`;

exports[`Utils formatter defaultKeyFormatter 142`] = `"cyan-10"`;

exports[`Utils formatter defaultKeyFormatter 143`] = `"blue-1"`;

exports[`Utils formatter defaultKeyFormatter 144`] = `"blue-2"`;

exports[`Utils formatter defaultKeyFormatter 145`] = `"blue-3"`;

exports[`Utils formatter defaultKeyFormatter 146`] = `"blue-4"`;

exports[`Utils formatter defaultKeyFormatter 147`] = `"blue-5"`;

exports[`Utils formatter defaultKeyFormatter 148`] = `"blue-6"`;

exports[`Utils formatter defaultKeyFormatter 149`] = `"blue-7"`;

exports[`Utils formatter defaultKeyFormatter 150`] = `"blue-8"`;

exports[`Utils formatter defaultKeyFormatter 151`] = `"blue-9"`;

exports[`Utils formatter defaultKeyFormatter 152`] = `"blue-10"`;

exports[`Utils formatter defaultKeyFormatter 153`] = `"arcoblue-1"`;

exports[`Utils formatter defaultKeyFormatter 154`] = `"arcoblue-2"`;

exports[`Utils formatter defaultKeyFormatter 155`] = `"arcoblue-3"`;

exports[`Utils formatter defaultKeyFormatter 156`] = `"arcoblue-4"`;

exports[`Utils formatter defaultKeyFormatter 157`] = `"arcoblue-5"`;

exports[`Utils formatter defaultKeyFormatter 158`] = `"arcoblue-6"`;

exports[`Utils formatter defaultKeyFormatter 159`] = `"arcoblue-7"`;

exports[`Utils formatter defaultKeyFormatter 160`] = `"arcoblue-8"`;

exports[`Utils formatter defaultKeyFormatter 161`] = `"arcoblue-9"`;

exports[`Utils formatter defaultKeyFormatter 162`] = `"arcoblue-10"`;

exports[`Utils formatter defaultKeyFormatter 163`] = `"purple-1"`;

exports[`Utils formatter defaultKeyFormatter 164`] = `"purple-2"`;

exports[`Utils formatter defaultKeyFormatter 165`] = `"purple-3"`;

exports[`Utils formatter defaultKeyFormatter 166`] = `"purple-4"`;

exports[`Utils formatter defaultKeyFormatter 167`] = `"purple-5"`;

exports[`Utils formatter defaultKeyFormatter 168`] = `"purple-6"`;

exports[`Utils formatter defaultKeyFormatter 169`] = `"purple-7"`;

exports[`Utils formatter defaultKeyFormatter 170`] = `"purple-8"`;

exports[`Utils formatter defaultKeyFormatter 171`] = `"purple-9"`;

exports[`Utils formatter defaultKeyFormatter 172`] = `"purple-10"`;

exports[`Utils formatter defaultKeyFormatter 173`] = `"pinkpurple-1"`;

exports[`Utils formatter defaultKeyFormatter 174`] = `"pinkpurple-2"`;

exports[`Utils formatter defaultKeyFormatter 175`] = `"pinkpurple-3"`;

exports[`Utils formatter defaultKeyFormatter 176`] = `"pinkpurple-4"`;

exports[`Utils formatter defaultKeyFormatter 177`] = `"pinkpurple-5"`;

exports[`Utils formatter defaultKeyFormatter 178`] = `"pinkpurple-6"`;

exports[`Utils formatter defaultKeyFormatter 179`] = `"pinkpurple-7"`;

exports[`Utils formatter defaultKeyFormatter 180`] = `"pinkpurple-8"`;

exports[`Utils formatter defaultKeyFormatter 181`] = `"pinkpurple-9"`;

exports[`Utils formatter defaultKeyFormatter 182`] = `"pinkpurple-10"`;

exports[`Utils formatter defaultKeyFormatter 183`] = `"magenta-1"`;

exports[`Utils formatter defaultKeyFormatter 184`] = `"magenta-2"`;

exports[`Utils formatter defaultKeyFormatter 185`] = `"magenta-3"`;

exports[`Utils formatter defaultKeyFormatter 186`] = `"magenta-4"`;

exports[`Utils formatter defaultKeyFormatter 187`] = `"magenta-5"`;

exports[`Utils formatter defaultKeyFormatter 188`] = `"magenta-6"`;

exports[`Utils formatter defaultKeyFormatter 189`] = `"magenta-7"`;

exports[`Utils formatter defaultKeyFormatter 190`] = `"magenta-8"`;

exports[`Utils formatter defaultKeyFormatter 191`] = `"magenta-9"`;

exports[`Utils formatter defaultKeyFormatter 192`] = `"magenta-10"`;

exports[`Utils formatter defaultKeyFormatter 193`] = `"gray-1"`;

exports[`Utils formatter defaultKeyFormatter 194`] = `"gray-2"`;

exports[`Utils formatter defaultKeyFormatter 195`] = `"gray-3"`;

exports[`Utils formatter defaultKeyFormatter 196`] = `"gray-4"`;

exports[`Utils formatter defaultKeyFormatter 197`] = `"gray-5"`;

exports[`Utils formatter defaultKeyFormatter 198`] = `"gray-6"`;

exports[`Utils formatter defaultKeyFormatter 199`] = `"gray-7"`;

exports[`Utils formatter defaultKeyFormatter 200`] = `"gray-8"`;

exports[`Utils formatter defaultKeyFormatter 201`] = `"gray-9"`;

exports[`Utils formatter defaultKeyFormatter 202`] = `"gray-10"`;

exports[`Utils formatter defaultKeyFormatter 203`] = `"success-1"`;

exports[`Utils formatter defaultKeyFormatter 204`] = `"success-2"`;

exports[`Utils formatter defaultKeyFormatter 205`] = `"success-3"`;

exports[`Utils formatter defaultKeyFormatter 206`] = `"success-4"`;

exports[`Utils formatter defaultKeyFormatter 207`] = `"success-5"`;

exports[`Utils formatter defaultKeyFormatter 208`] = `"success-6"`;

exports[`Utils formatter defaultKeyFormatter 209`] = `"success-7"`;

exports[`Utils formatter defaultKeyFormatter 210`] = `"success-8"`;

exports[`Utils formatter defaultKeyFormatter 211`] = `"success-9"`;

exports[`Utils formatter defaultKeyFormatter 212`] = `"success-10"`;

exports[`Utils formatter defaultKeyFormatter 213`] = `"primary-1"`;

exports[`Utils formatter defaultKeyFormatter 214`] = `"primary-2"`;

exports[`Utils formatter defaultKeyFormatter 215`] = `"primary-3"`;

exports[`Utils formatter defaultKeyFormatter 216`] = `"primary-4"`;

exports[`Utils formatter defaultKeyFormatter 217`] = `"primary-5"`;

exports[`Utils formatter defaultKeyFormatter 218`] = `"primary-6"`;

exports[`Utils formatter defaultKeyFormatter 219`] = `"primary-7"`;

exports[`Utils formatter defaultKeyFormatter 220`] = `"primary-8"`;

exports[`Utils formatter defaultKeyFormatter 221`] = `"primary-9"`;

exports[`Utils formatter defaultKeyFormatter 222`] = `"primary-10"`;

exports[`Utils formatter defaultKeyFormatter 223`] = `"danger-1"`;

exports[`Utils formatter defaultKeyFormatter 224`] = `"danger-2"`;

exports[`Utils formatter defaultKeyFormatter 225`] = `"danger-3"`;

exports[`Utils formatter defaultKeyFormatter 226`] = `"danger-4"`;

exports[`Utils formatter defaultKeyFormatter 227`] = `"danger-5"`;

exports[`Utils formatter defaultKeyFormatter 228`] = `"danger-6"`;

exports[`Utils formatter defaultKeyFormatter 229`] = `"danger-7"`;

exports[`Utils formatter defaultKeyFormatter 230`] = `"danger-8"`;

exports[`Utils formatter defaultKeyFormatter 231`] = `"danger-9"`;

exports[`Utils formatter defaultKeyFormatter 232`] = `"danger-10"`;

exports[`Utils formatter defaultKeyFormatter 233`] = `"warning-1"`;

exports[`Utils formatter defaultKeyFormatter 234`] = `"warning-2"`;

exports[`Utils formatter defaultKeyFormatter 235`] = `"warning-3"`;

exports[`Utils formatter defaultKeyFormatter 236`] = `"warning-4"`;

exports[`Utils formatter defaultKeyFormatter 237`] = `"warning-5"`;

exports[`Utils formatter defaultKeyFormatter 238`] = `"warning-6"`;

exports[`Utils formatter defaultKeyFormatter 239`] = `"warning-7"`;

exports[`Utils formatter defaultKeyFormatter 240`] = `"warning-8"`;

exports[`Utils formatter defaultKeyFormatter 241`] = `"warning-9"`;

exports[`Utils formatter defaultKeyFormatter 242`] = `"warning-10"`;

exports[`Utils formatter defaultKeyFormatter 243`] = `"link-1"`;

exports[`Utils formatter defaultKeyFormatter 244`] = `"link-2"`;

exports[`Utils formatter defaultKeyFormatter 245`] = `"link-3"`;

exports[`Utils formatter defaultKeyFormatter 246`] = `"link-4"`;

exports[`Utils formatter defaultKeyFormatter 247`] = `"link-5"`;

exports[`Utils formatter defaultKeyFormatter 248`] = `"link-6"`;

exports[`Utils formatter defaultKeyFormatter 249`] = `"link-7"`;

exports[`Utils formatter defaultKeyFormatter 250`] = `"link-8"`;

exports[`Utils formatter defaultKeyFormatter 251`] = `"link-9"`;

exports[`Utils formatter defaultKeyFormatter 252`] = `"link-10"`;
